ntbs-service-unit-tests/Services/NotificationServiceTest.cs

```diff
@@ -158,22 +152,21 @@ public void UpdateImmunosuppressionDetails_LeavesValuesUnchangedWhenStatusIsYes(
             };
             var notification = new Notification();
 
-            service.UpdateImmunosuppresionDetailsAsync(notification, input);
+            await service.UpdateImmunosuppresionDetailsAsync(notification, input);
 
             Assert.Equal(reference.Status, input.Status);
             Assert.Equal(reference.HasBioTherapy, input.HasBioTherapy);
             Assert.Equal(reference.HasTransplantation, input.HasTransplantation);
             Assert.Equal(reference.HasOther, input.HasOther);
             Assert.Equal(reference.OtherDescription, input.OtherDescription);
+            VerifyUpdateDatabase(mockContext);
         }
 
         [Theory]
         [InlineData((int)Status.No)]
         [InlineData((int)Status.Unknown)]
-        public void UpdateImmunosuppressionDetails_StripsAllButStatusWhenStatusIsNotYes(int status)
+        public async Task UpdateImmunosuppressionDetails_StripsAllButStatusWhenStatusIsNotYes(int status)
         {
-            // Hacky workaround to parameterise enum values seems to be required as passing enum directly resulted in console exception:
-            // System.IO.FileNotFoundException : Could not load file or assembly 'Microsoft.Extensions.Configuration.UserSecrets...

```

Comment:
Confused about this comment being removed, as no code is being removed along with it?
At the same time I'm not sure I understand the comment particularly well..? :P

Reply:
This was an issue with using an `enum` directly with `InlineData` - but since it's now done in 2 places I thought it might be example enough so people shouldn't stumble on it for too long in the future.
I can attempt to reword to make clearer and duplicate the comment in both instances if you would prefer?

Reply:
Ahh, OK!
Yeah, examples should be enough - though it might be worth moving the comment a readme in the test folder (troubleshooting section or whatnot)? so this knowledge isn't completely lost and the error is still searchable